backend/sirius-components-charts/src/main/java/org/eclipse/sirius/components/charts/piechart/elements/PieChartElementProps.java

```diff
@@ -94,12 +103,18 @@ public Builder keys(List<String> keys) {
             return this;
         }
 
+        public Builder style(PieChartStyle style) {
+            this.style = Objects.requireNonNull(style);
+            return this;
+        }
+
         public PieChartElementProps build() {
             PieChartElementProps pieChartElementProps = new PieChartElementProps();
             pieChartElementProps.id = Objects.requireNonNull(this.id);
             pieChartElementProps.descriptionId = Objects.requireNonNull(this.descriptionId);
             pieChartElementProps.values = Objects.requireNonNull(this.values);
             pieChartElementProps.keys = Objects.requireNonNull(this.keys);
+            pieChartElementProps.style = this.style; // Optional
```

Comment:
// Optional on purpose
(to be consistent with all other widgets)

Reply:
ok